### ISSUE TITLE
Definition lists not being treated as terms and definitions

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-ietf#118]
In IETF the following section gets generated wrongly. Check the image.

```adoc
== Terms and Definitions

Prompt::
JSON sent from the server to the client describing the tests the client performs

Registration::
The initial request from the client to the server describing the capabilities of one or several algorithm, mode and revision combinations

Response::
JSON sent from the client to the server in response to the prompt

Test Case::
An individual unit of work within a prompt or response

Test Group::
A collection of test cases that share similar properties within a prompt or response

Test Vector Set::
A collection of test groups under a specific algorithm, mode, and revision

Validation::
JSON sent from the server to the client that specifies the correctness of the response
```

![image](https://user-images.githubusercontent.com/11865/101438187-bf9d8000-394c-11eb-833f-440f75a53db0.png)

**Comment by opoudjis** [comment]
The fix is:

1. If the definitions list is retained, to introduce an empty boilerplate clause, to suppress the insertion of the "no terms" boilerplate
2. Preferred: to convert to proper terms markup. But suppressing the insertion of boilerplate is a legitimate requirement.

**Comment by ronaldtse** [comment]
Option 2 is better.

> 2. Preferred: to convert to proper terms markup. But suppressing the insertion of boilerplate is a legitimate requirement.

Agree.

**Comment by opoudjis** [comment]
So, option 2 is for whoever is marking up this document to do.

The options, as of https://github.com/metanorma/metanorma-ietf/issues/119, are:

1. Keep the terms markup as is, do not process it as a set of terms. Turn it into a boilerplate clause, which will also suppress the "No terms and definitions are listed in this document.":

````
== Terms and Definitions

[.boilerplate]
=== {blank}

Prompt::
JSON sent from the server to the client describing the tests the client performs
````

2. Update the terms markup to real terms markup. Currently that will result in subclauses for each term, because IETF does not know any better than to treat term clauses as clauses. Insert blank boilerplate to suppress the boilerplate "For the purposes of this document, the following terms and definitions apply.":

````
== Terms and Definitions

[.boilerplate]
=== {blank}

=== Prompt
JSON sent from the server to the client describing the tests the client performs

````

3. Update the terms markup to real terms markup. Currently that will result in subclauses for each term, because IETF does not know any better than to treat term clauses as clauses. Accept the Metanorma boilerplate boilerplate "For the purposes of this document, the following terms and definitions apply.":

````
== Terms and Definitions

=== Prompt
JSON sent from the server to the client describing the tests the client performs

````

4. Change IETF to not insert any terms boilerplate:

````
== Terms and Definitions

=== Prompt
JSON sent from the server to the client describing the tests the client performs

````

5. Change IETF to change terms clauses rendering into definition lists. (I do not want to do that, though admittedly something similar was done for NIST.)

**Comment by ronaldtse** [comment]
Can I ask for 6:

* Change IETF to not insert any terms boilerplate.
* Update the terms markup to real terms markup. 
* Allow custom content before the T&D listing.

```
== Terms and Definitions

MY CUSTOM CONTENT...

=== Prompt
JSON sent from the server to the client describing the tests the client performs
```

**Comment by opoudjis** [comment]
> Allow custom content before the T&D listing.

Addressed in #119
